cypress/integration/getTokenSilently.js

```diff
@@ -70,7 +91,7 @@ describe('getTokenSilently', function () {
 
       cy.login();
 
-      cy.route({
+      cy.intercept({
```

Comment:
`route` is deprecated in Cypress >= 6, and replaced by `intercept`, which works better in more situations. See https://docs.cypress.io/api/commands/intercept#Comparison-to-cy-route

I had to change it as the tests were failing.